jac-uk/apply
# #1952 Changes to Equality and diversity Form

## What's included?
Update the social mobility questions
Part of jac-uk/admin#1952

Closes jac-uk/admin#1952

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Visit the Equality and diversity part of the application form.
For an exercise opening after 1st April 2023 and you should see the new social mobility questions.
For an exercise opening before 1st April 2023 you should see the existing social mobility questions.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
